README.md

```diff
@@ -1,3 +1,9 @@
 # Learning course Modern Statistics for Modern Biology @ statistical genetics lab
 
 The course is basically on R, but we try to implment algorithms in other computer languages
```

Comment:
Would you fix my typo?

Reply:
Trying implementing 
http://www.tounou.net/wordpress/?p=226